v22.2/import-performance-best-practices.md

```diff
@@ -16,6 +16,14 @@ Import speed primarily depends on the amount of data that you want to import. Ho
 If the import size is small, then you do not need to do anything to optimize performance. In this case, the import should run quickly, regardless of the settings.
 {{site.data.alerts.end}}
 
+## TK section from IMPORT INTO doc .. manage import jobs?
```

Comment:
Is this still in progress?

Reply:
Didn't mean to include this in the PR! Thank you for catching. Reverting the changes.